l10n_it_vat_statement_split_payment/tests/test_vat_statement_split.py

```diff
@@ -252,17 +248,6 @@ def test_invoice(self):
         self.assertEqual(self.vat_statement.generic_vat_account_line_ids.amount, 22.0)
 
     def test_account_sp_company(self):
-        account_user_type = self.env.ref("account.data_account_type_receivable")
-        account_sp = self.account_model.create(
-            dict(
-                code="split_payment_acc",
-                name="Split payment account",
-                user_type_id=account_user_type.id,
-                reconcile=True,
-            )
-        )
-        self.company.sp_account_id = account_sp.id
-
```

Comment:
Questo non mi sembra essere necessario per la migrazione, puoi aprire una issue per riportare la correzione anche nelle altre versioni?
Per facilitare il porting alle altre versioni, sarebbe anche comodo avere la modifica in un commit dedicato.